draft-thomson-saag-sig-context.md

```diff
@@ -42,25 +42,35 @@ have different semantics.  This produces a potential problem whereby signatures
 with different intended uses can be confused.  The addition of context strings
 in signatures removes this problem.
 
+
 --- middle
 
 # Introduction
 
-There are a handful of cryptographic primitives that are used to build
-protocols that are used on the Internet: digital signatures, key derivation
-algorithms, message integrity codes, etc..  These protocols are generally
-developed in isolation from each other, with minimal effort to ensure that data
-structures used in one protocol do not have plausible interpretations in other
+Digital signatures are used in a range of different contexts.  These uses are
+often developed in isolation, which leads to the potential for data structures
```

Comment:
This reduces the scope to digital signatures, dropping key derivation algorithms and message integrity codes.  Do we want to narrow it in this way?